wrapper/wrapper.cpp

```diff
@@ -63,6 +63,9 @@ public ref struct DriverSettings
     [JsonProperty("Accel parameters")]
     Vec2<AccelArgs> args;
 
+    [JsonProperty("Output Speed Cap", Required = Required::Default)]
```

Comment:
We should probably put a warning or something around this new setting that it's not intended for most people